docs/integrations/digitalocean/k8s.mdx

```diff
@@ -0,0 +1,281 @@
+   ```
+
+1. You can now verify that the cluster is running successfully by looking at the pods in the new `ngrok-ingress-controller` namespace... wait, `CreateContainerConfigError`?
+
+   ```bash
+   kubectl get pods -n ngrok-ingress-controller
+
+   NAME                                                              READY   STATUS                       RESTARTS   AGE
+   ngrok-ingress-controller-kubernetes-ingress-controller-mannvs9b   0/1     CreateContainerConfigError   0          6m30s
+   ```
+
+   That's actually to be expected at this point. If you run `kubectl describe pods -n ngrok-ingress-controller YOUR_NGROK_INGRESS_CONTAINER`, you'd see the following error: `kubelet            Error: secret "ngrok-auth" not found`. Let's remedy that.
+
+1. Get your the `AUTHTOKEN` and `API_KEY`, which the ngrok Kubernetes Operator will use to authenticate against your existing ngrok account.
+
+   Find your `AUTHTOKEN` under [**Your Authtoken**](https://dashboard.ngrok.com/get-started/your-authtoken) in the ngrok dashboard.
+
+   To create a new API key, navigate to the [**API** section](https://dashboard.ngrok.com/api) of the ngrok dashboard, click the **New API Key** button, change the description or owner, and click the **Add API Key** button. Copy the API key token shown in the modal window before closing, as the ngrok dashboard will not show you the token again.
+
+1. While you're in the ngrok dashboard, create an ngrok static subdomain for ingress. Navigate to the [**Domains**
+   section](https://dashboard.ngrok.com/cloud-edge/domains) of the ngrok dashboard and click **Create Domain** or **New
+   Domain**. This static subdomain, like `example123.ngrok.app`, will be your `NGROK_DOMAIN` for the remainder of this guide.
+
+1. Create a file named `ngrok_creds.yaml` on your local workstation with the below YAML, replacing `API_KEY` and `AUTHTOKEN` but retaining the quotation (`"`) marks.
+
+   ```yaml
+   apiVersion: v1
+   kind: Secret
+   metadata:
+     name: ngrok-auth
+     namespace: ngrok-ingress-controller
+   stringData:
+     API_KEY: "NGROK_API_KEY"
+     AUTHTOKEN: "NGROK_AUTHTOKEN"
+   ```
+
+   The ngrok Kubernetes Operator pod should pick up on the new secret and authenticate properly.
+
+   Your new Kubernetes cluster, managed by DigitalOcean, is all set up—and with the ngrok Kubernetes Operator installed with a single option on the CLI.
+
+## **Step 2**: Deploy an example app on your DigitalOcean-managed cluster {#deploy-an-example-app}
+
+Let's explore how you can apps or services to your DigitalOcean-managed Kubernetes cluster. The DigitalOcean has a repository of [sample Kubernetes apps](https://github.com/digitalocean/kubernetes-sample-apps/tree/master/bookinfo-example), including the [Bookinfo](https://github.com/digitalocean/kubernetes-sample-apps/tree/master/bookinfo-example) app, which we'll use.
```

Comment:
"Let's explore who you can **add** apps..."

Is it "DigitalOcean" or "The DigitalOcean"? 🤔 I think it's "DigitalOcean has a repository..."